src/man/yaourtrc.5

```diff
@@ -129,6 +129,11 @@ Specify the folder to save temporary files (default is /tmp)\&.
 .RS 4
 Set to 1 to bypass the local database test\&.
 .RE
+.PP
+\fBCOMPRESS=1\fR
+.RS 4
+Set to 0 to disable compressing a built package before installing with pacman, for improved speed\&.
```

Comment:
English is not my native but I think the following would be more correct:
+Set to 0 to disable compressing of the package built before installing it with pacman, to improve speed\&.